common/Inc/ade7913_common.h

```diff
@@ -1,5 +1,7 @@
-#ifndef ADE7012_H_
-#define ADE7012_H_
+#ifndef ADE7013_COMMON_H
+#define ADE7013_COMMON_H
+
+#include "ade7913.h"
```

Comment:
@Justin-Vuong question: why have both "ade7913_common.h" and "ade7913.h" it seems "ade7913_common.h" isn't really needed since the same function declarations are found in "ade7913.h"